docs/user-guide/rules.md

```diff
@@ -142,6 +142,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`selector-no-universal`](../../src/rules/selector-no-universal/README.md): Disallow the universal selector.
 - [`selector-no-vendor-prefix`](../../src/rules/selector-no-vendor-prefix/README.md): Disallow vendor prefixes for selectors.
 - [`selector-pseudo-class-case`](../../src/rules/selector-pseudo-class-case/README.md): Specify lowercase or uppercase for pseudo-class selectors.
+- [`selector-pseudo-class-parentheses-space-inside`](../../src/rules/selector-parentheses-space-inside/README.md): Require a single space or disallow whitespace on the inside of the parentheses within selectors.
```

Comment:
"Require a single space or disallow whitespace on the inside of the parentheses within pseudo-class selectors."